CHANGELOG.md

```diff
@@ -1,3 +1,6 @@
+### Added
+- `log` method in Reporting API to send custom log messages with configurable log levels (predefined or custom). See [Reporting API methods](https://github.com/reportportal/agent-js-vitest?tab=readme-ov-file#log) for details.
+- `LOG_LEVELS` and `PREDEFINED_LOG_LEVELS` exports.
```

Comment:
Please remove this from here and from readme example - let's allow users to specify plain strings.